pkgs/tools/misc/lesspipe/default.nix

```diff
@@ -28,9 +46,61 @@ stdenv.mkDerivation rec {
   installFlags = [ "DESTDIR=$(out)" ];
 
   postInstall = ''
-    for f in lesspipe.sh lesscomplete; do
-      wrapProgram "$out/bin/$f" --prefix-each PATH : "${lib.makeBinPath [ file gnused procps ]}"
-    done
+    # resholve doesn't see strings in an array definition
```

Comment:
@abathur fyi

Reply:
https://github.com/wofr06/lesspipe/blob/63872f3214606e6e7d11616097cf6e3ffa54fc78/lesspipe.sh#L607 the specific case in context

Reply:
Speaking very broadly, this is out in the ~10% of dynamic behavior that is likely to remain outside of resholve's purview. 

We can either either tell resholve the value of the variable at execution time, or it has to have to be capable of deriving it from the script. 

In order to derive it from the script itself, it basically needs to a full shell execution model in order to figure out what the values should be. This can seem trivial in very narrow cases, but a general solution is actually pretty tricky--the package-time and run-time environments can differ in complex  ways. resholve would basically need to evaluate the script comprehensively enough to both know what the value should be, _and_ be 100% confident that the runtime environment couldn't affect it.

In other words, it's only possible to peel off very thin slices of genuinely dynamic behavior within resholve. It might be able to address functions that execute $N or $@ without any shifting or other variable-mutating shenanigans), but it gets tricky quickly beyond that point.